com.unity.netcode.gameobjects/Runtime/Components/NetworkAnimator.cs

```diff
@@ -1337,38 +1520,38 @@ private unsafe void SendParametersUpdateServerRpc(ParametersUpdateMessage parame
                     return;
                 }
 
-                m_ClientSendList.Clear();
+                m_TargetGroup.Clear();
                 foreach (var clientId in connectedClientIds)
                 {
-                    if (clientId == serverRpcParams.Receive.SenderClientId || clientId == NetworkManager.ServerClientId || !NetworkObject.Observers.Contains(clientId))
+                    if (clientId == rpcParams.Receive.SenderClientId || clientId == NetworkManager.ServerClientId || !NetworkObject.Observers.Contains(clientId))
                     {
                         continue;
                     }
-                    m_ClientSendList.Add(clientId);
+                    m_TargetGroup.Add(clientId);
                 }
 
-                m_ClientRpcParams.Send.TargetClientIds = m_ClientSendList;
-                m_NetworkAnimatorStateChangeHandler.SendParameterUpdate(parametersUpdate, m_ClientRpcParams);
+                m_RpcParams.Send.Target = m_TargetGroup;
+                m_NetworkAnimatorStateChangeHandler.SendParameterUpdate(parametersUpdate, m_RpcParams);
             }
         }
 
         /// <summary>
         /// Distributed Authority: Updates the client's animator's parameters
         /// </summary>
-        [Rpc(SendTo.NotAuthority)]
-        internal void SendParametersUpdateRpc(ParametersUpdateMessage parametersUpdate)
+        [Rpc(SendTo.NotAuthority, AllowTargetOverride = true, InvokePermission = RpcInvokePermission.Owner)]
```

Comment:
This will be changing the logic (might be intended). the default for `[Rpc(SendTo.NotAuthority)]` is `InvokePermission = RpcInvokePermission.Everyone`

Reply:
This just assures that in a distributed authority session (when this is only invoked) the owner/authority is the only instance that can send parameter updates (i.e. a more "secure" NetworkAnimator for distributed authority) to the non-authority instances.